### ISSUE TITLE
Error rendering code search page

### ISSUE THREAD

**Issue by ShayArtzi** [go-gitea/gitea#28580]
### Description

Hello,

Not sure when did this start, but I'm getting the following error when using code search:
```
An error occurred:

Render failed, failed to render template: explore/code, error: template error: builtin(bindata):code/searchresults:34:83 : executing "code/searchresults" at <Safe>: wrong type for value; expected string; got template.HTML
----------------------------------------------------------------------
								<td class="lines-code chroma"><code class="code-inner">{{.FormattedLines | Safe}}</code></td>
								                                                                           ^
----------------------------------------------------------------------
```
![image](https://github.com/go-gitea/gitea/assets/7936008/0ad49c09-e2ac-4767-985c-9debc8271891)

Looks like the search does work but rendering the results fails:
![image](https://github.com/go-gitea/gitea/assets/7936008/611f3357-9d38-4442-a461-e8abc06779ac)

I'm using `bleve` indexer with the following settings:
```
[indexer]
REPO_INDEXER_ENABLED = true
REPO_INDEXER_PATH = indexers/repos.bleve
MAX_FILE_SIZE = 1048576
REPO_INDEXER_INCLUDE =
REPO_INDEXER_EXCLUDE = resources/bin/**
```

The above config worked for me until I upgraded to one of the recent version (not sure which update, I haven't used code search for a bit).

I will appreciate any help - thanks very much!

### Gitea Version

1.21.3

### Can you reproduce the bug on the Gitea demo site?

No

### Operating System

ubuntu

### How are you running Gitea?

Running gitea binary as a systemd service on Ubuntu 22.04

### Database

MySQL/MariaDB

**Comment by lunny** [comment]
duplicated with #28575 ?

**Comment by wxiaoguang** [comment]
Please use 1.21 nightly build.

->  Fix 500 error of searching commits (#28576) #28579
